cli/add.go

```diff
@@ -0,0 +1,101 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/fishi0x01/vsh/client"
```

Comment:
CodeClimate complains that this import is redundant.

Reply:
Yeah, I'm not sure what it's on about there. It doesn't compile without it as client is used in the file.

Reply:
Interesting :thinking: 
Once you merge https://github.com/fishi0x01/vsh/pull/88 and rebase, it will evaluate again. Lets see what it says then. If it fails again, I will take a look

Reply:
Ok. Still failing. I will have a look :+1:

Reply:
Im not exactly sure what the issue here is. It seems to me that codeclimate uses a govet wrapper https://github.com/codeclimate-community/codeclimate-govet

Might be this is a bug. Running `go vet .` locally doesn't show any errors. I've marked the issue in CodeClimate as invalid